package.json

```diff
@@ -45,8 +45,10 @@
   },
   "dependencies": {
     "@material-ui/core": "3",
+    "@testing-library/react-hooks": "^3.2.1",
     "cozy-bar": "7.12.2",
     "cozy-client": "8.5.1",
+    "cozy-flags": "^2.2.0",
```

Comment:
Since all our other cozy package are not `^`, should we do the same here?

Reply:
Indeed! Fixed, thanks